src/observers/ErrorMessageObserver.ts

```diff
@@ -23,9 +22,15 @@ export class ErrorMessageObserver {
             case DotNetTestDebugStartFailure.name:
                 this.handleDotNetTestDebugStartFailure(<DotNetTestDebugStartFailure>event);
                 break;
+            case CorruptedDownloadError.name:
+                this.handleCorruptedDownloadError(<CorruptedDownloadError> event);
         }
     }
 
+    handleCorruptedDownloadError(event: CorruptedDownloadError): any {
```

Comment:
Constants seem to be duplicated?

Reply:
What are the constants you are referring to here ?